tomopy/misc/morph.py

```diff
@@ -73,7 +73,7 @@
 LIB_TOMOPY = extern.c_shared_lib('libtomopy')
 
 
-def pad(arr, axis, npad=None, mode='constant', **kwargs):
+def pad(arr, axis, npad=None, mode=str('constant'), **kwargs):
```

Comment:
So the unicode_literals import from future causes makes the mode kwarg be of type unicode, and this breaks the numpy pad function when using python 2.7
But I agree that this really should not be there...